### PR TITLE
New Post by Zemerik

### PR DESCRIPTION
**Pull request by Zemerik**
New Post: How to Code a Discord bot
Picture: Add my Profile Picture

### CODE REVIEW

File: src/content/posts/Discord bot.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+- Remember to replace **YOUR BOT TOKEN HERE** with the actual token of your bot which can be obtained from [the developer portal](https://discord.com/developers/applications)
+
+## 🥇Step 3 - Running your project
+
+- You are now ready to run your project. As soon as your run your project, your discord bot should appear online and send `pong` whenever someone types `ping`. 
+
+##
+
+<h4 align = "center">Thats all for this project. Hope you had fun while coding your discord bot and found this post helpful. /h4>
```
Comment (oyepriyansh) [inline src/content/posts/Discord bot.md:96]: close the h4 tag properly

Comment (oyepriyansh) [inline src/content/posts/Discord bot.md:96]: close the h4 tag properly

**Review by oyepriyansh — CHANGES_REQUESTED** [review]
image
The image name must be the same as your author name. Change it to `zemerik.png`.

**Review by oyepriyansh — APPROVED** [review]
LGTM

**Comment by Zemerik** [comment]
Thanks

**Pull request merged**